### PR TITLE
NoTask - Added /user/instagramcredentials endpoints

### PR DESCRIPTION
**Pull request by lgrzybowski**
I added Instragram API2 endpoint for BWAPI wrapper. 
I ran tests results looks like:

Finished in 0.02279 seconds (files took 0.56066 seconds to load)
41 examples, 0 failures

### CODE REVIEW

File: lib/bwapi/client/user.rb [hunk 1]
```diff
@@ -52,6 +53,7 @@ def ui_role
       end
 
       include BWAPI::Client::User::Notifications
+      include BWAPI::Client::User::InstagramCredentials
```
Comment (jonathanchrisp) [inline lib/bwapi/client/user.rb]: Really small change but we list all includes in alphabetical order to keep things neat and tidy

File: lib/bwapi/client/user/instagram_credentials.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        # Get current user Instagram credentials
+        #
+        # @return [Hash] current user Instagram credentials
+        def instagram_credentials
+          get '/user/instagramcredentials'
+        end
+
+        # Delete current user Instagram credentials
+        #
+        # @return [Hash] current user Instagram credentials and delete Instagram credentials
```
Comment (jonathanchrisp) [inline lib/bwapi/client/user/instagram_credentials.rb]: I think we can just put `current user Instagram credentials` here

File: lib/bwapi/client/user/instagram_credentials.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        def delete_instagram_credentials
+          delete '/user/instagramcredentials'
+        end
+
+        # Store the current user's Instagram credentials
+        #
+        # @option opts [String] accessToken Instagram API access token
+        # @option opts [String] clientId Instagram API client id
+        # @param opts [Hash] options Hash of parameters
```
Comment (jonathanchrisp) [inline lib/bwapi/client/user/instagram_credentials.rb]: Can we list out the options which can be passed in? You can see an example of this here: https://github.com/jonathanchrisp/bwapi/blob/master/lib/bwapi/client/projects.rb#L46

**Comment by jonathanchrisp** [comment]
@lgrzybowski - added initial comments

File: lib/bwapi/client/user/instagram_credentials.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+module BWAPI
+  class Client
+    module User
+      # Instagram module for user/instagramcredentials endpoints
```
Comment (benSlaughter) [inline lib/bwapi/client/user/instagram_credentials.rb]: The comment here needs to match the module name
`InstagramCredentials module for....`

File: lib/bwapi/client/user/instagram_credentials.rb (hunk 2 above)
Comment (benSlaughter) [inline lib/bwapi/client/user/instagram_credentials.rb]: as it's a delete endpoint would it make more sense to have `Current users deleted Instagram credentials` ?

File: lib/bwapi/client/user/instagram_credentials.rb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+        # Delete current user Instagram credentials
+        #
+        # @return [Hash] Current users deleted Instagram credentials
+        def delete_instagram_credentials
+          delete '/user/instagramcredentials'
+        end
+
+        # Store the current user's Instagram credentials
+        #
```
Comment (jonathanchrisp) [inline lib/bwapi/client/user/instagram_credentials.rb]: The opts line `@param opts [Hash] options Hash of parameters` should go above the parameters

**Pull request merged**